src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/mailer.html.twig

```diff
@@ -121,18 +121,18 @@
                                                 <h3 class="tab-title">Headers</h3>
                                                 <div class="tab-content">
                                                     <span class="label">Subject</span>
-                                                    <h2 class="m-t-10">{{ message.headers.get('subject').bodyAsString() ?? '(empty)' }}</h2>
+                                                    <h2 class="m-t-10">{{ message.getSubject() ?? '(empty)' }}</h2>
```

Comment:
should use `message.subject` here, as done elsewhere instead of explicitly calling the getter.

Reply:
same below

Reply:
Von: herberterich 10@gmail.com
herberterich10@ gmail.com>
Gesendet: Montag, 13. Februar 2023 09:46
An: Saniye Nesci <saniye.nesci@visamar.ch>
Betreff: Avis d'opéré
Madame, Monsieur,
Nous vous informons que nous avons effectué le virement détaillé ci-dessous :
Type de virement :
SEPA
Montant :
45,500.00 EUR
Date d'exécution :
13/02/2023
Référence de bout en bout :
1-47/2023
Motif :
urgent payement
Banque:
Visamar Group GmbH
Adresse:
Z Westferry Circus, E14 4HD, London, United Kingdom BIC:
CHASDEFX
Numéro de compte :
GB39-REVO-0099-6946-7599-54

Reply:
****

Reply:
That doesn't work:
```
Too few arguments to function Symfony\Component\Mime\Email::subject(), 0 passed in /home/egabor/work/closed-source/mentha/crewsader/vendor/twig/twig/src/Extension/CoreExtension.php on line 1635 and exactly 1 expected
```

Reply:
ah, indeed. I forgot that the setter does not have a `set` prefix in that case, making this incompatible with the Twig convention for getters